### ISSUE TITLE
Wrong variable scope in nested class

### ISSUE THREAD

**Issue by traveller59** [microsoft/pyright#989]
**Describe the bug**
pyright 1.1.65
statement in nested class can't access names defined in parent class, but pylance don't report any error or warning.

**To Reproduce**
```Python
class A:
    c = 5 
    class B:
        a = c
```
run ```pyright /path/to/this/code.py```, then run this file in python.

**Expected behavior**
pylance should indicate the 'c' is not defined.

**Screenshots or Code**
```Python
class A:
    c = 5 
    class B:
        a = c
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright and Pylance.

**Comment by erictraut** [comment]
This is now addressed in Pyright 1.1.66, which I just published. The change will also be in the next release of Pylance.
